images/kubernetesonarm/elasticsearch/run.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# provision elasticsearch user
+addgroup sudo
+adduser -D -g '' elasticsearch
+adduser elasticsearch sudo
+chown -R elasticsearch /elasticsearch /data
+echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+
+# set environment
+export CLUSTER_NAME=${CLUSTER_NAME:-elasticsearch-default}
+export NODE_MASTER=${NODE_MASTER:-true}
+export NODE_DATA=${NODE_DATA:-true}
+export HTTP_ENABLE=${HTTP_ENABLE:-true}
+export MULTICAST=${MULTICAST:-true}
+
+# Kubernetes stuff
+export NAMESPACE=${NAMESPACE:-default}
+export DISCOVERY_SERVICE=${DISCOVERY_SERVICE:-elasticsearch-discovery}
+
+# allow for memlock
+ulimit -l unlimited
+
+# run
+sudo -E -u elasticsearch /elasticsearch/bin/elasticsearch
```

Comment:
I'm not an expert at this, but why is `sudo` required?
I see you're adding an es user, but why not use it directly as `root`?